### PR TITLE
Story/ cord 31

### PR DESCRIPTION
**Pull request by aa-kashk**
App version needs to be shown in footer

### CODE REVIEW

File: cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+</head>
+<body>
+	<div th:fragment="footer" th:align="center">Cord19 Explorer version 2.0</div>
```
Comment (jdamerow) [inline cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html]: this needs to pull the version from the pom (similar to other apps like conceptpower).

**Review by jdamerow — CHANGES_REQUESTED** [review]
the footer insert needs to go into the layout not in every individual page

**Review by jdamerow — CHANGES_REQUESTED** [review]
The individual pages should not have the footer insert. That should be done in the layout.

**Comment by aa-kashk** [comment]
I have moved the footer and navbar insert to main.html. The individual pages do not have the footer insert anymore

**Pull request closed without merging**